base-transformer-class/transformer_class.py

```diff
@@ -214,7 +214,7 @@ def get_transformer_params(self, args: argparse.Namespace, metadata: list) -> di
                 parse_md = parse_md['pipeline']
             parsed_metadata.append(parse_md)
 
-            # Get the season, experiment, etc information
+            # Get the season, experiment.yml, etc information
```

Comment:
The global search/replace has incorrectly changed this here and elsewhere. Please corect